Library/Homebrew/utils/github.rb

```diff
@@ -487,7 +488,10 @@ def create_bump_pr(info, args:)
       changed_files += additional_files if additional_files.present?
 
       if args.dry_run? || (args.write? && !args.commit?)
-        ohai "try to fork repository with GitHub API" unless args.no_fork?
+        unless args.no_fork?
+          ohai "try to fork repository with GitHub API"
+          puts "..into `#{args.fork_org}` GitHub organization" if args.fork_org
```

Comment:
This still looks a bit weird (all lines are `ohai`s except this one, and the `..`):
```
...
==> brew audit --formula clair.rb
==> try to fork repository with GitHub API
..into `Blah` GitHub organization
==> git add /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/clair.rb
...
```

Would something this be okay?:
```suggestion
          fork_message = "try to fork repository with GitHub API#{" into `#{args.fork_org}` organization"}"
          ohai fork_message
```

Reply:
Done